appdev/guides/dependencies.rst

```diff
@@ -0,0 +1,136 @@
+Handle dependencies in C++ with CMake
+=====================================
+
+If an app depends on a library that is not pre-installed in Ubuntu Touch, the app needs to ship it inside the click package. This guide shows how this can be done with Clickable.
+
+Building
+--------
+Often libraries are available pre-built. So you need to decide, whether you want your build setup to download the library source and compile it or download the pre-built version. The following sections describe the options with its pros and cons.
+
+Compilation
+^^^^^^^^^^^
+Compiling the library gives you control over the result, which brings a lot of advantages. For instance, you may disable components your app doesn't need and link the library statically, resulting in a much smaller package size.
+
+Put the libraries source code at ``libs/LIBNAME``, because this is where clickable will look for it by default. If the dependency source code is available as a git repository, it is a good idea to add it as a `git submodule <https://git-scm.com/book/de/v1/Git-Tools-Submodule>`_. Otherwise you might want to add a script to download the sources.
+
+Add a `libraries section <http://clickable.bhdouglass.com/en/latest/clickable-json.html#libraries>`_ to your clickable.json, like this:
+
+.. code-block:: JSON
+   :emphasize-lines: 4-8
+
+    {
+      "template": "cmake",
+      "dir": "build/app",
+      "libraries": {
+        "LIBNAME": {
+          "template": "cmake"
+        }
+      }
+    }
+
+If the library does not contain a CMake configuration, you may use the `qmake` or `custom` template instead. If you choose `custom`, you need to add a `build` command.
+
+Note, that you should define a custom app build directory via the ``dir`` param as shown above. Otherwise a `clickable clean` (which is part of the default `clickable` command) would delete the library's build directory, too.
```

Comment:
I don't think "Note" is needed when it is used as the first word of the sentence anywhere in this document.

Reply:
Sorry for my bad English and thanks for your patience!